src/neptune/handler.py

```diff
@@ -538,32 +532,34 @@ def fetch(self):
             * `Namespace handler`
 
         Returns:
-            Value stored in the field or in case of a namespace a dictionary containing all non-Atom fields values.
+            Value stored in the field or, in case of a namespace, a dictionary containing all non-Atom fields values.
 
-        .. _Field types docs page:
+        For more information on field types, see the docs:
            https://docs.neptune.ai/api-reference/field-types
         """
         return self._pass_call_to_attr(function_name="fetch")
 
     def fetch_last(self):
-        """Fetches the last value stored in the series from Neptune servers.
+        """Fetches the last value stored in the series from Neptune.
+
+        Available for the following field types:
 
-        Available for following field types (`Field types docs page`_):
             * `FloatSeries`
             * `StringSeries`
 
         Returns:
-            Fetches the last value stored in the series from Neptune servers.
+            Fetches the last value stored in the series from Neptune.
 
-        .. _Field types docs page:
+        For more information on field types, see the docs:
            https://docs.neptune.ai/api-reference/field-types
         """
         return self._pass_call_to_attr(function_name="fetch_last")
 
     def fetch_values(self, *, include_timestamp: Optional[bool] = True):
-        """Fetches all values stored in the series from Neptune servers.
+        """Fetches all values stored in the series from Neptune.
+
+        Available for the following field types (`Field types docs page`_):
```

Comment:
Remove the reST markup

```suggestion
        Available for the following field types:
```